pkg/server/http/server.go

```diff
@@ -48,10 +50,8 @@ func Server(opts ...Option) (svc.Service, error) {
 		svc.Address(options.Config.HTTP.Addr),
 		svc.Context(options.Context),
 		svc.Flags(options.Flags...),
-		svc.Handler(applyMiddlewares(
-			options.Handler,
-			options.Middlewares...,
-		),
+		svc.Handler(
+			applyMiddlewares(options.Handler, options.Middlewares...)
```

Comment:
misses a trailing comma

Reply:
thanks! strange my linter didn't catch this